modules/ROOT/pages/cors-policy.adoc

```diff
@@ -4,24 +4,47 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cors, api gateway, gateway, policy
 
-Cross-Origin Resource Sharing (CORS) is a mechanism by which a web application can access resources defined in another domain. Browsers for example, rely on this standard for allowing cross origin requests.
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | cross-origin resource sharing (CORS)
+>s| Summary      | Enables access to resources residing in external domains.
+>s| Category | Compliance
+>s| Available from Mule version | v4.1.1
+.4+>.^s| Returned Status Codes |
 
-The CORS policy complies with Cross-Origin Resource Sharing W3C Recommendation 16 January 2014. From API Manager, you can apply the policy, the default policy configuration considers the API to be a public resource. If not, removing the Public Resource check will enter the extended CORS configuration.
+CORS is a mechanism by which a Web application can access resources that are defined in another domain.  Using CORS, applications invoking JavaScript XMLHttpRequest (XHR) calls from a Web page can interact with resources from non-origin domains.
+
+Typically, Web browsers implement this standard for allowing cross origin requests. When you apply this policy to your API, it is considered a public resource by default. You can  check will enter the extended CORS configuration. If require that your API be shared with only specific resources, you can configure the policy accordingly.
```

Comment:
This sentence seems incomplete: "You can  check will enter the extended CORS configuration"

Reply:
Done! Thanks!